app/models/message.rb

```diff
@@ -3,7 +3,8 @@ class Message < ApplicationRecord
   belongs_to :trip
 
   validates_presence_of :trip, :sender_name, :sender_email, :body
-
+  validates_format_of :sender_email, with: /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
```

Comment:
We could add the Regex in a constant in `ApplicationRecord` or create a custom validator to not repeat the code from `trip.rb`.